keyboards/handwired/dactyl_manuform_carbonfet/4x5/keymaps/default/keymap.c

```diff
@@ -0,0 +1,72 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _BASE,
+    _RAISE,
+    _LOWER
```

Comment:
clang-format likes it when you have a comma at the end, like this.  For those that do want to run a format pass, this is a small sanity saver.  ;)

```suggestion
    _LOWER,
```